index.js

```diff
@@ -49,10 +68,31 @@ getUsersCreditDetails(usersArray);
 // ***************************************************************************
 
 const genderView = users => {
-  // Your code goes here ...
+  const femaleUsers = [];
+  const maleUsers = [];
+
+  let femaleTotal = users.filter((female) => {
+    return femaleTotal.gender === "female";
+  });
+  
+  for(let {firstName, lastName} of femaleTotal){
+    femaleUsers.push(`${firstName} ${lastName}`)
+  };
+
+  let maleTotal = users.filter((male) => {
+    return maleTotal.gender === "male";
+  });
+  
+  for(let {firstName, lastName} of maleTotal){
+    maleUsers.push(`${firstName} ${lastName}`)
+  };
+  let gendersObj = {};
+  gendersObj.femaleUsers = femaleUsers;
+  gendersObj.maleUsers = maleUsers;
+  return gendersObj;
```

Comment:
Good job!
You could have, once again, made it all in less steps:
either:
```js
let gendersObj = {
femaleUsers,
maleUsers
}
return gendersObj;
```
☝️ you are making less steps.

But:
Even faster:
```js
return {
femaleUsers,
maleUsers
}
)
```